inputfiles/overridingTypes.jsonc

```diff
@@ -218,6 +218,22 @@
                         }
                     }
                 }
+            },
+            "WindowOrWorkerGlobalScope": {
+                "methods": {
+                    "method": {
+                        "clearInterval": {
+                            "overrideSignatures": [
+                                "clearInterval(id: number | undefined): void"
```

Comment:
`overrideSignatures` should be used as a last resort. Could you do the `signature` way as it's done below in OnErrorEventHandlerNonNull etc.?